### DISCUSSION TITLE
Checking for initial major/minor updates

### DISCUSSION THREAD

**Discussion by p3k** [renovatebot/renovate#27151] [Suggest an Idea]
### Tell us more.

This is a follow-up to https://github.com/renovatebot/renovate/discussions/27119:

I would like to add a label to versions like 1.0.0 or 2.1.0, i.e. initial releases of a major or minor version.

This should make them a tad more prominent because such versions often still contain bugs and might need more thorough testing.

I know about matchCurrentVersion and matchCurrentValue but it seems there is no “matchUpdateVersion” or something similar…

@rarkins is positive about the idea:

> I'm open to a matchNewValue option for packageRules

Here is a hypothetical repo with the details how it could work: https://gitlab.com/repros2/renovate-match-new-version

**Comment by rarkins** [comment]
Feature request issue: https://github.com/renovatebot/renovate/issues/27313
